.github/workflows/release-merge.yml

```
@@ -0,0 +1,30 @@
+name: Create PR on Release
+
+# Triggered when a new version is released publicly
+on:
+  release:
+    types: [released]
+
+jobs:
+  create-pull-request:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        id: checkout
+        # https://github.com/actions/checkout/releases/tag/v4.1.6
+        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29
+
+      - name: Create Pull Request
+        id: create_pull
+        env:
+          GH_TOKEN: ${{ secrets.RELEASE_TOKEN }}
+        run: |
+          # Building the PR Data
+          PR_DATA=$(gh pr create \
+            --title "Merge release into master" \
```

In the [github docs](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) we can see that `github.ref` has this observation:

> [...] For workflows triggered by release, this is the release tag created. [...]

And `github.ref_name` is the shorthand tag name, which means we can use:

```suggestion
            --title "[${{ github.ref_name }}] merge release into master" \
```

So that the name of the PR is `[v1.1.1]: merge release into master` or some other title that includes the release version.

---

That's a great suggestion! Added on https://github.com/HathorNetwork/hathor-wallet/pull/626/commits/af582a2280245bdecc038c7139bf49546a5f824d